### PR TITLE
CustomersInsertUpdateDelete

### PR DESCRIPTION
**Pull request by Yanetta**

### CODE REVIEW

**Review by Artsapollo — COMMENTED** [review]
Looks good
But why did you add two customers and deleted only one of them?

File: src/test/java/CustomersDaoImplTest.java [hunk 1]
```diff
@@ -1,28 +1,81 @@
...
+    @Test
+    public void test1InsertCustomer() throws SQLException {
+        System.out.println("testInsertCustomer ____________________");
+        boolean ins = customersDao.insertCustomer(BigDecimal.valueOf(3333), BigDecimal.valueOf(333));
+        System.out.println(ins);
+        assertTrue(ins);
+    }
+
+    @Test
+    public void test2InsertCustomer1() throws SQLException {
```
Comment (Artsapollo) [inline src/test/java/CustomersDaoImplTest.java]: Also I think it's much easier to put all parameters in assertTrue as one line

File: src/main/java/workwithJDBC/CustomersDao.java [hunk 2]
```diff
@@ -7,5 +7,16 @@
 
 public interface CustomersDao {
     List<Customers> getAllCustomers() throws SQLException;
+
     List<Customers> findCustomersByCreditLimit(BigDecimal creditLimit) throws SQLException;
+
+    boolean insertCustomer(BigDecimal cust_num, BigDecimal credit_limit) throws SQLException;
+
+    boolean insertCustomer1(Customers customer) throws SQLException;
```
Comment (SerhiiPasko) [inline src/main/java/workwithJDBC/CustomersDao.java]: bad method name

File: src/main/java/workwithJDBC/CustomersDao.java [hunk 3]
```diff
@@ -7,5 +7,16 @@
...
+
+    boolean updateCustomer(BigDecimal cust_num) throws SQLException;
+
+    boolean updateCustomer1(Customers customer, BigDecimal cust_num) throws SQLException;
```
Comment (SerhiiPasko) [inline src/main/java/workwithJDBC/CustomersDao.java]: also

File: src/main/java/workwithJDBC/CustomersDaoImpl.java [hunk 4]
```diff
@@ -50,4 +50,160 @@
...
+        PreparedStatement stmt = null;
+        Boolean isRowInserted = false;
+        try {
+            connection = Pool.getConnection();
+            stmt = connection.prepareStatement("INSERT INTO customers (cust_num, credit_limit) VALUES (?, ?)");
+            stmt.setBigDecimal(1, cust_num);
+            stmt.setBigDecimal(2, credit_limit);
+            isRowInserted = stmt.executeUpdate() > 0;
+
+        } catch (SQLException e) {
```
Comment (SerhiiPasko) [inline src/main/java/workwithJDBC/CustomersDaoImpl.java:67]: in this case maybe make thence throw SQLException on method level

File: src/main/java/workwithJDBC/CustomersDaoImpl.java [hunk 5]
```diff
@@ -50,4 +50,160 @@
...
+            e.printStackTrace();
+        } finally {
+            try {
```
Comment (SerhiiPasko) [inline src/main/java/workwithJDBC/CustomersDaoImpl.java:70]: if you use it one block try..catch

**Review by SerhiiPasko — APPROVED** [review]
Read comments. Good

**Review by Artsapollo — APPROVED** [review]

**Pull request merged**